draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
The issue I see is that even if you make the plugins a list a plugin might change it's behaviour and add or remove decorators. I wouldn't recommend it, but I can see this being useful for advanced editors. So in fact even if we compare the plugins the decorators might have changed.

You started to make me think in another direction.

```js
import Editor, { decoratorFromPlugins } from 'draft-js-plugins-editor';
import createImagePlugin from 'draft-js-image-plugin';
import createToolbarPlugin from 'draft-js-toolbar-plugin';
import { EditorState } from 'draft-js';

const imagePlugin = createImagePlugin();
const toolbarPlugin = createToolbarPlugin();
const plugins = [imagePlugin, toolbarPlugin];
const decorator = decoratorFromPlugins(plugins);

const editorState = EditorState.createEmpty(decorator);
```